### ISSUE TITLE
Tabs with dropdown and angular-permission support

### ISSUE THREAD

**Issue by itailulu** [rpocklin/ui-router-tabs#86]
Is it possible to have tabs with dropdowns [like here](http://getbootstrap.com/components/#tabs-with-dropdowns)?
Also, does ui-router-tabs support the use permissions on states (via angular-permission, [which support ui-router](https://github.com/Narzerus/angular-permission/wiki/Usage-in-ui-router-states))

Thanks!

**Comment by rob-pocklington-a139759** [comment]
Yes, tabs and pill are supported, use `type: [ 'tabs' | 'pills' ]`.  Permissions on states, to allow selection or display?  Selection disabling can by done by setting `disable: true` in the `tabData` tab you want disabled.  To hide/show currently this is not an option, but could easily be added.
